README.md

```diff
@@ -3,14 +3,55 @@
 [![GitHub Build Status](https://github.com/cisagov/skeleton-action-composite/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-action-composite/actions)
 
 This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub project started.
-This skeleton project contains [licensing information](LICENSE), as
-well as [pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
+new [cisagov](https://github.com/cisagov) GitHub composite Action
```

Comment:
What do you think about adding a link to https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action or https://docs.github.com/en/actions/sharing-automations/creating-actions/about-custom-actions#composite-actions here?

Reply:
Please see commit 26a14dc.

Reply:
Thanks!